tests/ops/qubit/test_parametric_ops.py

```diff
@@ -996,7 +1002,7 @@ def test_pcphase(self, phi, dim, wires):
         mat2 = op.compute_matrix(*op.parameters, **op.hyperparameters)
 
         expected_mat = np.diag(
-            [np.exp(1j * phi) if i < dim else 1.0 for i in range(2**num_wires)]
+            [np.exp(1j * phi) if i < dim else np.exp(-1j * phi) for i in range(2**num_wires)]
```

Comment:
How this is computed? If computed by hand or it is a trivial solution, might add a comment to clarify.

Reply:
Its trivial, the result is by definition of the operator